README.md

```diff
@@ -1,9 +1,14 @@
-# Cookiecutter Data Science
+# Cookiecutter Data Science GDS
 
 _A logical, reasonably standardized, but flexible project structure for doing and sharing data science work._
 
 
-#### [Project homepage](http://drivendata.github.io/cookiecutter-data-science/)
+#### [Project homepage](http://ukgovdatascience.github.io/cookiecutter-data-science-gds/)
+
+
+### Acknowledgment:
+-----------
+This project builds on [drivendata's cookiecutter-data-science project template](http://drivendata.github.io/cookiecutter-data-science/) #cookiecutterdatascience
 
 
 ### Requirements to use the cookiecutter template:
```

Comment:
Haven't checked with Python 2.7, but with Python 3.5 the tests all fail! Need to amend the next few lines and suggest using Python 3.6 or above.

Failure is to do with using an outdated library `joblib` instead of `joblib2` - using the old library fails the entire test suite.

Reply:
I've edited the lines and also modified the `{{ cookiecutter.python_interpreter }}` options (restricted to "python3" only) and the `test_enviornment.py` file accordingly, requiring python 3.6 or above.  

Still need to modify `Makefile` accordingly, also depending on whether we want to leave `conda` as default environment manager and/or we want to add `pipenv`.